CPS-0002/README.md

```diff
@@ -0,0 +1,93 @@
+---
+CPS: 2
+Title: Coin Selection Approach for Post Native Tokens Era in Cardano
```

Comment:
```suggestion
Title: Coin Selection Including Native Tokens
```
If author & editors are in agreement that this is more specific and less redundant ("era", "Cardano") then let's please change the document title to the current PR title.

Reply:
p.s. also @SIDANWhatever for fixing little things like this we recommend that your PR branch be set to "Allow edits by maintainers" (the general default when submitting a PR, and helpful to keep track of who recommended the changes behind each commit).

Reply:
> p.s. also @SIDANWhatever for fixing little things like this we recommend that your PR branch be set to "Allow edits by maintainers" (the general default when submitting a PR, and helpful to keep track of who recommended the changes behind each commit).

Seems its only work for PR from feature branch, should I close this PR and create a new one?

Reply:
no need to create a new PR... you should be able to extend the permissions by following these instructions (i.e. when you view this PR's branch in the repository which pushed the pull request) 🤓 https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests